charts/ccd-api-gateway-web/values.preview.template.yaml

```diff
@@ -5,4 +5,5 @@ nodejs:
     IDAM_OAUTH2_TOKEN_ENDPOINT: https://idam-api.aat.platform.hmcts.net/oauth2/token
     IDAM_OAUTH2_LOGOUT_ENDPOINT: https://idam-api.aat.platform.hmcts.net/session/:token
     IDAM_BASE_URL: https://idam-api.aat.platform.hmcts.net
+    CORS_ORIGIN_WHITELIST: "*"
```

Comment:
Why are we allowing any domain in the CORS whitelist? Isn't it contrary to the purpose of the fix attempt?